### DISCUSSION TITLE
Video quality

### DISCUSSION THREAD

**Discussion by pszafer** [Arkady23/V380-Web#3] [Q&A]
Is it possible that with your firmware patch enabled video quality is lower?
I have impression that video quality is lower than before applying this patch.

**Comment by Arkady23** [comment]
You can turn everything off. Disable telnet and disable writing to the sd-card vs app. Check the quality. Basically, the processor is loaded with RTSP and Record to the sd-card. But you can also disable http, but then, to enable it, you will have to remove the sd-card and use the card-reader to edit the file startup.sh by removing the "#" character from the first position in the httpd startup command.

**Reply by pszafer** [reply]
can I disable recording only?

**Reply by Arkady23** [reply]
In the V380 pro app, which I have installed on my phone, there is a disable permanent recording, disable event recording, and disable audio recording. I am currently doing a web-server update, where there will be a cloud disconnect and communication with the app.
P.S.
The app also has the ability to record to the sd-card not HD quality, but SD. This also greatly affects the CPU load.
